src/create-glamorous.js

```diff
@@ -149,6 +149,13 @@ function createGlamorous(splitProps) {
         },
       })
 
+      function withComponent(newComp, options) {
+        return glamorous(newComp, {
+          forwardProps: GlamorousComponent.forwardProps,
+          ...options,
```

Comment:
Makes sense to me. I defaulted it anyway just to illustrate intent (that it can be optionally provided) :+1: